docs/getting-started/ios/fastlane-swift.md

```diff
@@ -0,0 +1,39 @@
+# Getting Started with Fastlane.swift
```

Comment:
lowercase "F"? :)

Reply:
this follows swift conventions 😄

Reply:
doesn't fastlane convention trump Swift convention??

Reply:
I dont think so, the API that is generated is a file called `Fastlane.swift` I think we should stick with that convention, IMO.

Reply:
Shouldn't it be called `Fastfile.swift` though? Or am I getting things mixed up?

If this is just a name for this new feature/functionality, why not call it `fastlane Swift`?

Reply:
It's referring to the `Fastlane.swift` file that @ohayon mentioned above:

<img width="232" alt="screen shot 2017-12-12 at 9 23 58 am" src="https://user-images.githubusercontent.com/626066/33889494-3c702eb6-df1e-11e7-8603-743dffa9f35d.png">